doc/source/faq.rst

```diff
@@ -369,3 +369,151 @@ just a thin layer around the ``QTableView``.
 	    mw = MainWidget()
 	    mw.show()
 	    app.exec_()
+
+
+
+.. _ref-subclassing-pandas:
+
+Subclassing pandas Data Structures
+----------------------------------
+
+This section describes how to subclass ``pandas`` data structures to meet more specific needs. There are 2 points to be cared:
```

Comment:
I would add a note here, something like:

In general you should prefer to use *composition* rather than *inheritance* with pandas data structures., see [here](http://en.wikipedia.org/wiki/Composition_over_inheritance).